swarm/network/stream/snapshot_sync_test.go

```diff
@@ -92,6 +93,15 @@ func TestSyncingViaGlobalSync(t *testing.T) {
 		if *longrunning {
 			chnkCnt = []int{1, 8, 32, 256, 1024}
 			nodeCnt = []int{16, 32, 64, 128, 256}
+		} else if raceTest {
```

Comment:
is this still needed on top of using db vs in memory store?

Reply:
Yes, it is.